architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -105,11 +104,6 @@ installation documentation] provides specific examples using the `native` HA met
 configuring HAProxy. You can also take the concepts and apply them towards your
 existing HA solutions using the `native` method instead of HAProxy.
 
-[NOTE]
```

Comment:
Do we need to remove this note? Scott said that you can have single masters, so I think this is still relevant information.

Reply:
Yes, this can be removed, you can scale up masters now. You probably couldn't when this was originally written.

Reply:
This should be fine for 3.7+ don't bother with anything older than 3.7.